### PR TITLE
Update integrationtest.md instructions now that all signals are supported

### PR DESCRIPTION
**Pull request by aabmass**

### CODE REVIEW

File: exporter/collector/integrationtests.md [hunk 1]
```diff
@@ -2,18 +2,23 @@
...
 can be written to the real GCP APIs. Separately, the tests also mock the real APIs to capture
-and compare requests against recorded expectation fixtures. Currently, these tests only support
-metrics.
+and compare requests against recorded expectation fixtures. You can write tests for metrics,
+traces, and logs.
 
-The fixtures are located in the [`integrationtest/testdata/fixtures`](integrationtest/testdata/fixtures) directory and registered in
-[`integrationtest/testcases.go`][testcases].
+<a id="testcasefiles"></a>The fixtures are located in the [`integrationtest/testdata/fixtures`](integrationtest/testdata/fixtures) directories and registered in
+signal dependent test case files:
```
Comment (psx95) [inline exporter/collector/integrationtests.md:9]: NIt: could be in a single line

File: exporter/collector/integrationtests.md [hunk 2]
```diff
@@ -56,17 +62,20 @@ go test -tags=integrationtest -run=TestIntegration
...
 
-To add a new test:
+To add a new test, using metrics as an example:
 
-1. Create an OTLP fixture. Currently, the tests only support metrics fixtures which should be
-    JSON encoded OTLP
-    [`ExportMetricsServiceRequest`](https://github.com/open-telemetry/opentelemetry-proto/blob/b43e9b18b76abf3ee040164b55b9c355217151f3/opentelemetry/proto/collector/metrics/v1/metrics_service.proto#L35)
-    message. Put the fixture in the [`testdata/fixtures`](integrationtest/testdata/fixtures) directory. As an example,
-    see
+1. Create an OTLP fixture. The test fixtures should be JSON encoded OTLP messages of type:
```
Comment (psx95) [inline exporter/collector/integrationtests.md:67]: Nit: should we use permalinks for protos here ?

**Review by psx95 — COMMENTED** [review]
The content LGTM. 

Just have a small question about the anchor link.

File: exporter/collector/integrationtests.md [hunk 3]
```diff
@@ -84,7 +92,7 @@ To add a new test:
 
     This is how the Ops Agent fixtures were generated.
 
-1. Add an entry in [`integrationtest/testcases.go`][testcases].
+1. Add an entry in the corresponding [testcase file](#testcasefiles). For metrics, that is [`integrationtest/testcases/testcases_metrics.go`][testcases].
```
[the remark this answers is not in the document]
Comment (damemi) [reply exporter/collector/integrationtests.md:96]: it links to the `<a id="testcasefiles"></a>` on line 8

File: exporter/collector/integrationtests.md (hunk 2 above)
Comment (aabmass) [reply exporter/collector/integrationtests.md:67]: There are permalinks for tag v0.19.0

**Review by psx95 — APPROVED** [review]

**Pull request merged**